builtin/providers/aws/resource_aws_elb.go

```diff
@@ -418,6 +446,30 @@ func resourceAwsElbUpdate(d *schema.ResourceData, meta interface{}) error {
 			},
 		}
 
+		logs := d.Get("access_logs").(*schema.Set).List()
+		if len(logs) > 1 {
+			return fmt.Errorf("Only one access logs config per ELB is supported")
+		} else if len(logs) == 1 {
+			log := logs[0].(map[string]interface{})
+			accessLogs := &elb.AccessLog{
```

Comment:
Just a tiny nit: sort of odd that this local var is plural since we're only making one.